addons/sale_stock/models/sale_order.py

```diff
@@ -101,9 +101,20 @@ def create(self, values):
     def write(self, values):
         lines = False
         if 'product_uom_qty' in values:
-            precision = self.env['decimal.precision'].precision_get('Product Unit of Measure')
-            lines = self.filtered(
-                lambda r: r.state == 'sale' and float_compare(r.product_uom_qty, values['product_uom_qty'], precision_digits=precision) == -1)
+            self.ensure_one()
```

Comment:
Maybe iterate over the lines that bulk update of ```product_uom_qty``` is made possible even if I think this is never used in core and not very smart from a functional point of view.